tests/spread/plugins/catkin/catkin-with-python/task.yaml

```diff
@@ -22,5 +22,9 @@ restore: |
 execute: |
   cd "$SNAP_DIR"
 
+  # rosinstall files shouldn't require git to be installed, and this test is only
+  # completely valid if it ISN'T installed
+  apt remove --yes git || true
```

Comment:
should be apt-get

Reply:
Don't they do the same thing in this case?

Reply:
They use different backend implementations and we pollute the logs with the "do not use in scripts" message

Reply:
Ah that makes sense, thank you for explaining. Fixed.